src/work-allocation/containers/task-action/task-action-container.component.ts

```diff
@@ -60,12 +61,15 @@ export class TaskActionContainerComponent implements OnInit {
     };
 
     // Get the task from the route, which will have been put there by the resolver.
-    const { task } = this.route.snapshot.data.task;
+    const { task } = this.route.snapshot.data.taskAndCaseworkers.task;
     this.tasks = [ task ];
     this.routeData = this.route.snapshot.data as RouteData;
     if (!this.routeData.actionTitle) {
       this.routeData.actionTitle = `${this.routeData.verb} task`;
     }
+    if (task.assignee) {
+      this.caseWorkerService.getAll().subscribe(caseworkers => task.assigneeName = getAssigneeName(caseworkers, task.assignee));
```

Comment:
I thought `caseworkers` was now coming from `this.route.snapshot.data.taskAndCaseworkers`...?

Reply:
yes changed it. now